### DISCUSSION TITLE
Dialog box to confirm reaction

### DISCUSSION THREAD

**Discussion by lukeburns** [fonsp/Pluto.jl#1269] [Questions about using Pluto]
I have an expensive / time-intensive cell that I don't always want to react to other cells. I'd like a confirmation dialog that asks me if I want to run the cell when it reacts. I'm imagining something like:

```jl
if @js("confirm('Run cell?')")
   # run expensive simulation
end
```

`@bind` isn't a particularly good solution for this, because it requires first adding an element to the dom before Pluto can listen to its events. (e.g. you can't access a bound variable within the same cell it is bound).

**Comment by disberd** [comment]
I have found myself in a similar situation, but usually I'd prefer to set a checkbox or something similar to manually trigger when reactivity should be active or not.
If this kind of confirmation box would be triggered everytime an related cell is updated I suppose it might get distracting and break the seameless reactivity.

Is there some specific reason you'd prefer a confirmation dialog to a manual toggle?
